tasks/51_abbreviated-name/abbreviated-name.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * abbreviated - Converts a fullname in abbreviated form.
+ *
+ * @example
+ * //return 'Verbovyi D. O.'
+ * abbreviated('Verbovyi Dmytro Oleksandrovych');
+ * @param {String} str - fullname which function will convert.
+ * @return {String} abbreviated form of source string.
+ */
+export function abbreviated(str) {
+  return str.split(' ').reduce((acc, cur, index) => {
```

Comment:
Is it necessary to use `reduce()` here? From my perspective, you could use just `for` that starts from index `1`. Don't you think so?